Snowflake.Data.Tests/SFDbTransactionIT.cs

```diff
@@ -68,7 +68,7 @@ public void TestTransactionRollback()
             conn.Open();
 
             DbCommand command = conn.CreateCommand();
-            command.CommandText = "create or replace table testTransaction(x timestamp, a integer)";
+            command.CommandText = "create or replace table testTransaction(x timestamp_ntz, a integer)";
```

Comment:
Why change the data type?

Reply:
If account's TIMESTAMP_TYPE_MAPPING is set to TIMESTAMP_LTZ then in testTransation TIMESTAMP is treated as TIMESTAMP_LTZ and reader.GetDateTime throws an exception. Of course we can (a) set session's TIMESTAMP_TYPE_MAPPING to TIMESTAMP_NTZ or (b) change account's parameter. But as we don't test here TIMESTAMP_* variations I think explicit creation of TIMESTAMP_NTZ is the simplest solution.